tests/runtest.sh

```diff
@@ -1025,6 +1059,9 @@ do
         --no-lf-conversion)
             noLFConversion=ON
             ;;
+        --build-overlay-only)
+            buildOverlayOnly=ON
```

Comment:
I guess this switch should be incompatible with coreOverlayDir. It's not a big deal, but you may want to call it out in the help text and/or print an error and exit if both are specified.